tests/acceptance/features/apiCors/cors.feature

```diff
@@ -0,0 +1,66 @@
+@api @skipOnOcV10
+Feature: CORS headers
```

Comment:
add please description: if you want to run test locally - you should run ocis with env "OCIS_CORS_ALLOW_ORIGINS": "https://aphno.badal"